src/javascript/registrations/groups/registerRoutes.js

```diff
@@ -4,9 +4,9 @@ import React from 'react';
 
 export const registerRoutes = function () {
     registry.add('adminRoute', 'manageGroups', {
-        targets: ['administration-server-usersAndRoles:1'],
+        targets: ['administration-server-usersAndRoles:20'],
         requiredPermission: 'adminGroups',
-        icon: <Group/>,
+        icon: null,
```

Comment:
We need to remove Group from import as well, otherwise we can have problems building the module due to eslint issues

Reply:
No, because this same file also registers a group site settings for Adminstration/Sites (look at line 18 of the file), where the icon is needed. But in Administration/Server, we shall not have an icon.